src/mountscript.sh

```diff
@@ -196,9 +200,43 @@ search_free_local_ip_with_prefix()
     fi
     
     local local_ip=""
+    local optimize_get_free_local_ip=false
+    IFS=$'\n' 
```

Comment:
Which newly added command needs IFS?
I can't see anything that needs this. If not needed, let's remove it.